packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts

```diff
@@ -250,6 +250,8 @@ export const messages = {
     'sfdx-project.json ファイル (%s) をパースできませんでした。パースエラー: %s',
   sfdx_cli_not_found:
     'Salesforce CLI がインストールされていません。[%s](%s) からインストールしてください。',
+  sfdx_cli_not_supported:
+    '!!! Your installed CLI version is no longer supported. Please uninstall your CLI and reinstall the latest version using this link: [%s](%s)',
```

Comment:
This is a new error message.  The English version is in i18n.ts but I don't have a Japanese translation so I temporarily copied it over and added 3 exclamation points in front. @sbudhirajadoc